experimental/sql/src/impl_to_iterators.py

```diff
@@ -149,7 +157,7 @@ def match_and_rewrite(self, op: RelImpl.IndexByName,
 class CompareRewriter(RelImplRewriter):
 
   def convert_comparator(self, comparator: str) -> int:
-    if comparator == "==":
+    if comparator == "=":
```

Comment:
Naughty SQL only uses one `=` ;)

Reply:
Yep, this was code I had previously copied from ChocoPy, since we used a similar approach there :sweat_smile: